config/routes.rb

```diff
@@ -3,4 +3,16 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+#  root "mains#routes"
+  root "mains#index"
+  resources :users
+  resources :mains
+  resources :restaurants
```

Comment:
Do we need all of these routes?

Reply:
yes mam , we have some actions regarding that. and we can declare it in same line also